### ISSUE TITLE
Country flag emojis are just 2 letter abbreviations of country names. Much better if they were proper country flags.

### ISSUE THREAD

**Issue by selivan** [laurent22/joplin#9505]
### Operating system

Windows

### Joplin version

2.13.9

### Desktop version info

Joplin 2.13.9 (prod, win32)

Client ID: 3c3b2cb8e2194f7791b36b2260e2fae2
Sync Version: 3
Profile Version: 44
Keychain Supported: Yes

Revision: 1bbec44

Conflict Resolution: 1.2.3
Copy Code Blocks: 1.1.0
Note list and sidebar toggle buttons: 1.0.3

### Current behaviour

![image](https://github.com/laurent22/joplin/assets/1208989/f0e6dca8-9c8f-4d44-a64d-061887fdeac3)

Emoji flags inserted into notes also turn into 2-letter abbreviations.

### Expected behaviour

![image](https://github.com/laurent22/joplin/assets/1208989/a3a053ce-6b21-48f3-a772-4301f2112e52)

**Comment by personalizedrefrigerator** [comment]
Related: https://answers.microsoft.com/en-us/windows/forum/all/flag-emoji/85b163bc-786a-4918-9042-763ccf4b6c05 (found via [this StackOverflow post](https://stackoverflow.com/questions/62729729/how-do-i-view-country-flags-on-windows-10-through-html)).

Note that they do show up as flag emojis on Ubuntu 23.10:
<img alt="screenshot: Emoji picker with flags" src="https://github.com/laurent22/joplin/assets/46334387/0095ba1d-aa91-4b54-9433-4cb3ad4a194e" width="200"/>

**Comment by selivan** [comment]
@personalizedrefrigerator
If that is the case, I suppose Joplin should have an option to choose emoji font: system or some built-in options.

It sucks that Windows still does not include proper emoji set by default, but it is not a good reason to look at ugly letter flags.

Besides, Android app displays proper flag emojis, so user gets different experience on different platforms.

For example, Telegram client, from which I copied picture with proper flag emojis, offer these options:
![image](https://github.com/laurent22/joplin/assets/1208989/038343c8-56e9-4446-8355-9e0074f5d910)

**Comment by laurent22** [comment]
There's always the option to use an actual image - there are open source libraries of flag icons you could use, and then you're sure it would render the same on all platforms.

**Comment by selivan** [comment]
@laurent22 That is a workaround for notebook icons. But that will not work for emojis inside notes.

**Comment by laurent22** [comment]
Right, that's only for the notebooks. I guess we could bundle our own emoji lib like WhatsApp does but I'm not sure there's enough demand for it. Perhaps something to investigate

**Comment by selivan** [comment]
Besides messengers, like Whatsup and Telegram, Firefox and Thunderbird on Windows also include custom emoji, because bundled with system suck. It could be worth looking into, improves user experience.
